docs/how-to-contribute-to-apache-carbondata.md

```diff
@@ -55,35 +61,44 @@ For moderate or large contributions, you should not start coding or writing a de
 there is a corresponding JIRA issue assigned to you for that work. Simple changes,
 like fixing typos, do not require an associated issue.
 
-### Design
+
+
+
+
+## Design
```

Comment:
why change Design with H2 header(markdown)

Reply:
According to this line (No.29): "Engage -> Design -> Code -> Review -> Commit", the headings (Engage, Design, Code, Review) should be in same level, which is heading level 2.